lib/inc/internal/facts/resolvers/operating_system_resolver.hpp

```diff
@@ -25,6 +25,14 @@ namespace facter { namespace facts { namespace resolvers {
          */
         virtual void resolve(collection& facts) override;
 
+        /**
+         * Parses the major and minor OS release versions for Linux distros.
+         * @param name The name of the OS.
+         * @param release The release to parse.
+         * @return Returns a tuple of major and minor release versions.
+         */
+        static std::tuple<std::string, std::string> parse_distro(std::string const& name, std::string const& release);
```

Comment:
Never mind, this makes sense to do since it's parsing the "linux-specific" part of the structured fact.